src/ray/common/id.h

```diff
@@ -150,6 +149,12 @@ std::ostream &operator<<(std::ostream &os, const ObjectID &id);
 // Restore the compiler alignment to defult (8 bytes).
 #pragma pack(pop)
 
+/// Compute the root task ID of a driver.
+///
+/// \param job_id The job ID associated to the driver.
+/// \return The root task ID for the driver.
+TaskID ComputeDriverTaskId(const JobID &job_id);
```

Comment:
Hmm do you think this should be a `static` method of `TaskID`? That did sort of make sense to me.

Reply:
Thanks, I restored it!